src/Controls/src/Core/View.cs

```diff
@@ -199,5 +224,13 @@ void ValidateGesture(IGestureRecognizer gesture)
 			if (gesture is PinchGestureRecognizer && _gestureRecognizers.GetGesturesFor<PinchGestureRecognizer>().Count() > 1)
 				throw new InvalidOperationException($"Only one {nameof(PinchGestureRecognizer)} per view is allowed");
 		}
+
+		override protected internal void ChangeVisualState()
+		{
+			if (IsPointerOver)
```

Comment:
I'd say just move this to `VisualElement` where the other checks are and that `Disabled` should take precedence over `IsPointerOver`

That's how the ordering works for WinUI so we should just stay consistent